rllib/algorithms/ddpg/tests/test_ddpg.py

```diff
@@ -138,8 +144,8 @@ def test_ddpg_loss_function(self):
         config.replay_buffer_config = {
             "type": "MultiAgentReplayBuffer",
             "capacity": 50000,
-            "learning_starts": 0,
         }
+        config.num_steps_sampled_before_learning_starts = 0
```

Comment:
same thing here. you have already specified num_steps_sampled_before_learning_starts in line 133

Reply:
**TODO**

Reply:
Fixed!